xbmc/cores/VideoPlayer/DVDCodecs/Video/AddonVideoCodec.cpp

```diff
@@ -198,8 +198,6 @@ CDVDVideoCodec::VCReturn CAddonVideoCodec::GetPicture(VideoPicture* pVideoPictur
     pVideoPicture->iHeight = picture.height;
     pVideoPicture->pts = static_cast<double>(picture.pts);
     pVideoPicture->dts = DVD_NOPTS_VALUE;
-    pVideoPicture->color_range = 0;
```

Comment:
decoders are supposed to set these values explicitly. relying on arbitrary pre-set values is error prone